app/controllers/welcome_controller.rb

```diff
@@ -28,6 +29,17 @@ def mentor
     @report = params[:report].blank? ? "basics" : ERB::Util.html_escape(params[:report])
   end
 
+  def participant
+    if @participant && !@participant.completed_intake_form?
```

Comment:
When is `@participant` initialized? It looks like redirection will never happen because `@participant` will always be `nil`